src/views/virtualmachinesinstance/list/VirtualMachinesInstancesRow.tsx

```diff
@@ -28,7 +30,8 @@ const VirtualMachinesInstancesRow: React.FC<VirtualMachinesInstancesRowProps> =
         <VMStatusConditionLabelList conditions={obj?.status?.conditions?.filter((c) => c.reason)} />
       </TableData>
       <TableData id="created" activeColumnIDs={activeColumnIDs}>
-        {obj?.metadata?.creationTimestamp}
+        <GlobeAmericasIcon />{' '}
+        {format(new Date(obj?.metadata?.creationTimestamp), 'MMM dd, yyyy, h:mm a')}
```

Comment:
nice :) 
can you please add a tooltip that contains UTC time format as we have in 4.10?

Reply:
I don't see it in vmi list. It's there and I'm missing it?

Reply:
we will replace this all this components (in all the lists) with a common component from the component repo, no need for this one to be perfect